Fix invalid version in generated rofi.pc:

    usr/lib/pkgconfig/rofi.pc: pkgconf version 1.7.0-wayland is invalid

--- a/meson.build
+++ b/meson.build
@@ -322,7 +322,7 @@
 pkg.generate(
     filebase: 'rofi',
     name: 'rofi',
-    version: meson.project_version(),
+    version: meson.project_version().replace('-wayland', ''),
     description: 'Header files for rofi plugins',
     variables: [
         'pluginsdir=@0@'.format(join_paths('${libdir}', meson.project_name())),
